source/admin_manual/sasl.rst

```diff
@@ -8,7 +8,7 @@ SASL stands for "Simple Authentication and Security Layer". SASL itself is
 nothing more than a list of requirements for
 :ref:`authentication-authentication_mechanisms` and protocols to be
 SASL-compatible as described in
-`RFC 4422 <http://tools.ietf.org/html/rfc4422>`_. IMAP, POP3, SMTP, and
+:rfc:`4422`. IMAP, POP3, SMTP, and
 ManageSieve protocols all have support for SASL.
```

Comment:
After removing such a long part of the line you could re-flow the text, i.e. join the lines as far as possible, respecting the line-length.